conf/config.yaml

```diff
@@ -48,6 +48,7 @@ apisix:
     - 223.5.5.5
     - 1.1.1.1
     - 8.8.8.8
+  dns_resolver_valid: 30          # valid time for dns result 30 seconds
```

Comment:
I am not get your point why we need this PR, I think you should explain bug in PR

Reply:
https://github.com/apache/incubator-apisix/pull/1077#issue-364080437

I added some description right now, please take a look.